### DISCUSSION TITLE
送我去维修点的司机笑了十分钟

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#775] [General]
![image](https://user-images.githubusercontent.com/9006264/161371699-3a46e16c-596e-4f7c-9395-c210886d73a5.jpeg)

**Comment by Arondight** [comment]
我想起来前几年我有个电脑主板嗝屁了，过保了问了下维修要自费换主板，然后小广告上打电话，找了个会修的小哥，查了好几个小时找出来是有个电容打穿了，主板还没有烧掉，270块钱给我搞好了，还特意跟我说这零件四块钱，剩下的都是手艺费……如果不是🍎电脑，修理这块如果会的话成本还是挺低的

**Reply by mark9804** [reply]
我是苹果自己搞出来的那个触摸条坏了，我觉得应该是intel CPU太热把T2芯片整脱焊了，但是主板保修四年啊，我就很自信的去了分销店（全日本直营店就几家，只能分销了），结果他说你这个机子啊，主板保修四年，键盘也保修四年，但是给你单换Touch Bar不保修，掏钱吧，人民币4200
![image](https://user-images.githubusercontent.com/9006264/161376340-4370da40-53aa-4898-8bd2-bedeb6a2ab04.jpeg)

**Reply by Arondight** [reply]
让我就不修了，反正这个都被砍掉了

**Reply by mark9804** [reply]
肯定不修，根本问题没解决修了也还会继续坏……但是不修没 esc 键用了，属实缺大德
